src/modules/github/components/pull-review-form/utils.test.ts

```diff
@@ -0,0 +1,114 @@
+import { getCheckedPullsInfo } from '@/components/github/root/pull-review-form/utils'
+
+import type { TRepoHasCheck } from '@/types/github/root/index'
+
+describe('getCheckedPullsInfo', () => {
+  it('return array of checkedPulls', () => {
+    const repoHasCheckArray: TRepoHasCheck[] = [
+      {
+        org: 'org1',
+        repo: 'repo1',
+        pulls: [
+          {
+            title: 'pull1',
+            number: 1,
+            isChecked: true,
+            user: { login: 'user1' }
+          },
+          {
+            title: 'pull2',
+            number: 2,
+            isChecked: false,
+            user: { login: 'user2' }
+          },
+          {
+            title: 'pull3',
+            number: 3,
+            isChecked: true,
+            user: { login: 'user3' }
+          }
+        ]
+      }
+    ]
+
+    const result = getCheckedPullsInfo(repoHasCheckArray)
+
+    expect(result).toEqual([
+      { org: 'org1', repo: 'repo1', pullTitle: 'pull1', pullNumber: 1 },
+      { org: 'org1', repo: 'repo1', pullTitle: 'pull3', pullNumber: 3 }
+    ])
+  })
+  it('return arrray of that isChecked is null', () => {
+    const repoHasCheckArray: TRepoHasCheck[] = [
+      {
+        org: 'org1',
+        repo: 'repo1',
+        pulls: [
+          {
+            title: 'pull1',
+            number: 1,
+            isChecked: false,
+            user: { login: 'user1' }
+          },
+          {
+            title: 'pull2',
+            number: 2,
+            isChecked: false,
+            user: { login: 'user2' }
+          }
+        ]
+      }
+    ]
+
+    const result = getCheckedPullsInfo(repoHasCheckArray)
+
+    expect(result).toEqual([])
+  })
+  it('return arrray of that repo > 0', () => {
+    const repoHasCheckArray: TRepoHasCheck[] = [
+      {
+        org: 'org1',
+        repo: 'repo1',
+        pulls: [
+          {
+            title: 'pull1-1',
+            number: 1 - 1,
+            isChecked: true,
+            user: { login: 'user1-1' }
+          },
+          {
+            title: 'pull1-2',
+            number: 1 - 2,
+            isChecked: false,
+            user: { login: 'user1-2' }
+          }
+        ]
+      },
+      {
+        org: 'org1',
+        repo: 'repo2',
+        pulls: [
+          {
+            title: 'pull2-1',
+            number: 2 - 1,
+            isChecked: true,
+            user: { login: 'user2-1' }
+          },
+          {
+            title: 'pull2-2',
+            number: 2 - 2,
+            isChecked: false,
+            user: { login: 'user2-2' }
+          }
+        ]
+      }
+    ]
+
+    const result = getCheckedPullsInfo(repoHasCheckArray)
+
+    expect(result).toEqual([
+      { org: 'org1', repo: 'repo1', pullTitle: 'pull1-1', pullNumber: 1 - 1 },
+      { org: 'org1', repo: 'repo2', pullTitle: 'pull2-1', pullNumber: 2 - 1 }
+    ])
+  })
```

Comment:
1. for the `pullNumber`, it seems so weird. just put the regular number.
2. it('should return checked pulls information', ...)